Marlin/language_fr.h

```diff
@@ -35,12 +35,17 @@
 #define WELCOME_MSG                         MACHINE_NAME " prete."
 #define MSG_SD_INSERTED                     "Carte inseree"
 #define MSG_SD_REMOVED                      "Carte retiree"
+#define MSG_LCD_ENDSTOPS                    "Butées" // Max length 8 characters
 #define MSG_MAIN                            "Menu principal"
 #define MSG_AUTOSTART                       "Demarrage auto"
 #define MSG_DISABLE_STEPPERS                "Arreter moteurs"
 #define MSG_AUTO_HOME                       "Origine auto."
+#define MSG_AUTO_HOME_X                     "Origine X Auto."
+#define MSG_AUTO_HOME_Y                     "Origine Y Auto."
+#define MSG_AUTO_HOME_Z                     "Origine Z Auto."
 #define MSG_LEVEL_BED_HOMING                "Origine XYZ"
 #define MSG_LEVEL_BED_WAITING               "Cliquer pour commencer"
```

Comment:
Some strings are too long. This would be a good time to fix them.
```
12345678901234567890
Cliquer pour commenc|er
Regler decal. origin|e
Err: TEMP. MAX PLATE|AU
Plateau en chauffe..|.
```